modules/m_testline.c

```diff
@@ -235,7 +235,8 @@ mo_testline(struct MsgBuf *msgbuf_p, struct Client *client_p, struct Client *sou
 		sendto_one_numeric(source_p, RPL_STATSILINE, form_str(RPL_STATSILINE),
 				aconf->info.name, EmptyString(aconf->spasswd) ? "<NULL>" : aconf->spasswd,
 				show_iline_prefix(source_p, aconf, aconf->user),
-				aconf->host, aconf->port, aconf->className);
+				aconf->host, aconf->port, aconf->className,
+				EmptyString(aconf->desc) ? "" : aconf->desc);
```

Comment:
Why not reuse `CheckEmpty()` here?

Reply:
First because I didn't know about it at first and then second because it wasn't already in scope and I was avoiding adding new includes, but it's probably better to just use it.